app/src/ui/history/selected-commit.tsx

```diff
@@ -193,7 +193,7 @@ export class SelectedCommit extends React.Component<
   private onHideWhitespaceInDiffChanged = async (
```

Comment:
Maybe this function should change but I sorta think not 🤔

Reply:
Ditto